cmd/zdb/zdb.c

```diff
@@ -4423,7 +4423,6 @@ main(int argc, char **argv)
 			dump_opt[c] += verbose;
 	}
 
-	aok = (dump_opt['A'] == 1) || (dump_opt['A'] > 2);
```

Comment:
Rather than removing `aok` how about implementing it.  This way we can support the `zdb -A` option documented in the man.  Right now it's silently ignored.  This global is used [here in OpenZFS](https://github.com/openzfs/openzfs/blob/master/usr/src/uts/common/os/printf.c#L323) and would need to be added [here](https://github.com/zfsonlinux/zfs/blob/master/lib/libspl/include/assert.h#L36) for Linux.

```
     -A      Do not abort should any assertion fail.

     -AA     Enable panic recovery, certain errors which would otherwise be
             fatal are demoted to warnings.

     -AAA    Do not abort if asserts fail and also enable panic recovery.
```

Reply:
@behlendorf 
Sure, I'll try to implement it and send it in another PR.
Feel free to close this one.